python/GafferSceneUI/_InspectorColumn.py

```diff
@@ -176,18 +177,57 @@ def __disablableInspectionTweaks( pathListing ) :
 
 			with inspectionContext :
 				inspection = column.inspector().inspect()
-				if inspection is not None and inspection.canDisableEdit() :
-					tweaks.append( inspection )
+				if inspection is None :
+					return [], True
+
+				if inspection.canDisableEdit() :
+					enabledTweaks.append( inspection )
 				else :
-					return []
+					edit = inspection.acquireEdit( createIfNecessary = False )
+					if isinstance( edit, ( Gaffer.NameValuePlug, Gaffer.OptionalValuePlug, Gaffer.TweakPlug ) ) and Gaffer.Metadata.value( edit, "inspector:disabledEdit" ) :
+						disabledTweaks.append( inspection )
+					else :
+						# The selection includes a cell that is not viable candidate for enabling or disabling,
+						# we don't allow toggling of any of the selected cells in this case.
+						return [], True
```

Comment:
Oh, looks like you've changed the "silently ignore the others" behaviour I was questioning earlier. That's good I think. But it seems we've thrown away information about why something couldn't be disabled by doing that on this branch. Ideally we'd surface as much of that information to the user as possible when they hit `D` and nothing happens. And perhaps we could also include it in a tooltip for a greyed-out "Disable Edit" menu item?

Reply:
Now surfacing as much info as I have to hand as part of the overall messaging improvements in 1968cbf.

Including a tooltip on a menu item via `"description"` exposed an issue where we weren't hiding the tooltip when the cursor moved to a menu item without one, so I've addressed that separately in d09d2a6.